components/Books/CatchLog/CatchLog.js

```diff
@@ -43,8 +43,8 @@ export default function CatchLog() {
     const { id } = c;
     const { species, weight, length, spot_name, lure, cloudinary_urls } =
       c.attributes;
-
-    const image = (
+      
+    const image = cloudinary_urls.length === 0 ? null : (
```

Comment:
I think what it might be is the assignation of the empty cloudinary url (in the case of a missing image) to the image variable Image source on line 50 is what may be throwing the error. Maybe it can't assign it at all and that's what's triggering the error. That doesn't seem right though because it says it was trying to render it and couldn't but the conditional rendering logic on line 64 of the original code should be stopping it from rendering.